### ISSUE TITLE
Committees plan

### ISSUE THREAD

**Issue by jfer-rubio** [wizeline/FE.X#4]
What committees are going to be available
What metrics/outcomes we want to get out of them
How to add new committees or topics

define rules:
max people
common interest
quality?
owners?

**Comment by jfer-rubio** [comment]
[workflow doc](https://docs.google.com/document/d/1q4xSIW9sUowKV0FqSRTGgJ9NlmuRexn9X3ORHf27o04/edit?usp=sharing)

**Comment by memorammstein** [comment]
what's the difference between `trustees` and `committees`?

**Comment by jfer-rubio** [comment]
its a synonym but seems like not really working :(

**Comment by jfer-rubio** [comment]
https://docs.google.com/document/d/1q4xSIW9sUowKV0FqSRTGgJ9NlmuRexn9X3ORHf27o04/edit?usp=sharing

**Comment by salvadorbfm** [comment]
Since we're starting we should pick a couple of committees and work with that.

**Comment by jfer-rubio** [comment]
@salvadorbfm lets do that, we already have the design system going on. I think the best way of doing this is that whoever is now at the @wizeline/fe-x team starts a committee

**Comment by TheMightyPenguin** [comment]
I was thinking of a `Creative coding` (maybe another name) committee that would do something like:
* See actual use cases of new technology
* How to use cutting edge features to make amazing features
* How to use WebGL to build an amazing (and memorable) experience for the end users

When talking of creative coding, I think of stuff like these:
* http://edankwan.com/experiments/the-spirit/
* http://activetheory.net/ (these guys rock)
And just stuff that people do web technology that amazes the users.

What do you think about this committee? Would it make sense? or should it be part of another committee?

@salvadorbfm @jfer-rubio ^

**Comment by charliesbox** [comment]
@TheMightyPenguin yeah! we were thinking in a similar use case. We called it "Front End Labs"

We will go back to the basics and compare the barebones of the points you explained and the ones we have to evolve the idea.

Thanks!

**Comment by TheMightyPenguin** [comment]
@jfer-rubio just checked that doc 👍. It'd say it's clear enough as an starting point. Also, maybe adding info how to join a comittee? Not sure if that should go there or somewhere else.

**Comment by jfer-rubio** [comment]
@TheMightyPenguin thanks for checking it. There should be definitely the place to add that information. We don't know how joining committees will happen. while we deice that I'll add that section into that document.

**Comment by TheMightyPenguin** [comment]
Would be glad to be part of that (Front End Labs) committee once this gets started 😄!

**Comment by jfer-rubio** [comment]
Count on that!
